src/integrationTest/java/com/hivemq/helmcharts/CompatibilityHelmChartDeploymentIT.java

```diff
@@ -0,0 +1,60 @@
+package com.hivemq.helmcharts;
+
+import com.hivemq.client.mqtt.MqttGlobalPublishFilter;
+import com.hivemq.client.mqtt.datatypes.MqttQos;
+import com.hivemq.client.mqtt.mqtt5.Mqtt5BlockingClient;
+import com.hivemq.client.mqtt.mqtt5.Mqtt5Client;
+import com.hivemq.client.mqtt.mqtt5.message.publish.Mqtt5Publish;
+import com.hivemq.helmcharts.util.OperatorHelmChartContainer;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test that the chart is deployed successfully on specific kubernetes cluster versions
+ */
+@Testcontainers
+public class CompatibilityHelmChartDeploymentIT {
```

Comment:
Please be more consistent when using var or explicit type declarations. Either one or the other.